### PR TITLE
Remove compliance and visibility reporters

### LINKED ISSUE #241: Doc is wrong - https://github.com/chef-cookbooks/audit/blob/master/docs/supported_configuration.md
Report method "Report to Automate via Chef Server"

chef-server-automate does not need the automate url in chef-server.rb since in this scenario we integrated automate with chef server.

```Report to Automate via Chef Server

['audit']['reporter'] = 'chef-server-automate'
['audit']['server'] = 'https://compliance-server.test/api'
['audit']['refresh_token' OR 'token'] = '..'
['audit']['owner'] = 'User/Org'
#
# chef-server.rb:
data_collector['root_url'] = 'https://automate-server.test/data-collector/v0/'
```

### Cookbook version
[Version of the cookbook where you are encountering the issue]

### Chef-client version
[Version of chef-client in your environment]

### Platform Details
[Operating system distribution and release version. Cloud provider if running in the cloud]

### Scenario:
[What you are trying to achieve and you can't?]

### Steps to Reproduce:
[If you are filing an issue what are the things we need to do in order to repro your problem? How are you using this cookbook or any resources it includes?]

### Expected Result:
[What are you expecting to happen as the consequence of above reproduction steps?]

### Actual Result:
[What actually happens after the reproduction steps? Include the error output or a link to a gist if possible.]

### PR DESCRIPTION
**Pull request by alexpop**
Fixes #241

@vjeffrey @kekaichinose, should be also cleanup any doc references to the integration with the original Compliance product?

### CODE REVIEW

**Comment by kekaichinose** [comment]
@alexpop per #241, yeah, I would update this doc to clean up the reference:
https://github.com/chef-cookbooks/audit/blob/master/docs/supported_configuration.md

Thanks!

**Comment by vjeffrey** [comment]
ya, +1 on the cleanup, the original compliance has been eol for a while now

File: docs/supported_configuration.md [hunk 1]
```diff
@@ -1,145 +1,96 @@
...
-  </td>
-</tr>
-<tr>
-  <th>Fetch From Compliance via Chef Server</th>
-  <td><b>Report Directly to Compliance</b>
-<pre lang="ruby"><code>
-['audit']['reporter'] = 'chef-compliance'
-['audit']['fetcher'] = 'chef-server'
-['audit']['server'] = 'https://compliance-server.test/api'
-['audit']['refresh_token' OR 'token'] = '..'
```
Comment (vjeffrey) [inline docs/supported_configuration.md]: removing all this cruft is going to make this SO much more readable 🎉

**Comment by alexpop** [comment]
Merging this now as https://github.com/chef-cookbooks/audit/pull/385 went in as well

**Pull request merged**